.circleci/dependencies.txt

```diff
@@ -0,0 +1,232 @@
+perl-Lingua-EN-Inflect-1.903
+perl-List-MoreUtils-0.428
+perl-List-MoreUtils-XS-0.428
+perl-local-lib-2.000023
+perl-Log-Contextual-0.008001
+perl-LWP-MediaTypes-6.02
+perl-Meta-Builder-0.003
+perl-Minion-9.13
+perl-Minion-Backend-SQLite-4.005
+perl-Module-Build-0.422400
+perl-Module-Find-0.13
+perl-Module-Implementation-0.09
+perl-Module-Pluggable-5.2
+perl-Module-Runtime-0.016
+perl-Module-Runtime-Conflicts-0.003
+perl-Mojo-IOLoop-ReadWriteProcess-0.23
+perl-Mojolicious-8.24
+perl-Mojolicious-Plugin-AssetPack-2.08
+perl-Mojo-Pg-4.16
+perl-Mojo-RabbitMQ-Client-0.3.1
+perl-Mojo-SQLite-3.002
+perl-Moo-2.003004
+perl-Moose-2.2010
+perl-MooseX-Role-Parameterized-1.10
+perl-MRO-Compat-0.13
+perl-multidimensional-0.014
+perl-namespace-autoclean-0.28
+perl-namespace-clean-0.27
+perl-Net-AMQP-0.06
+perl-Net-HTTP-6.17
+perl-Net-IP-1.26
+perl-Net-OpenID-Common-1.20
+perl-Net-OpenID-Consumer-1.18
+perl-Net-SNMP-6.0.1
+perl-Net-SSH2-0.69
+perl-Net-SSLeay-1.81
+perl-Package-DeprecationManager-0.17
+perl-Package-Stash-0.37
+perl-Package-Stash-XS-0.28
+perl-Package-Variant-1.003002
+perl-PadWalker-2.3
+perl-Params-Util-1.07
+perl-Params-Validate-1.29
+perl-Params-ValidationCompiler-0.26
+perl-Parse-PMFile-0.41
+perl-Parse-RecDescent-1.967015
+perl-Path-Class-0.37
+perl-Path-Tiny-0.104
+perl-Perl-Critic-1.130
+perl-Perl-Critic-Freenode-0.026
+perl-Perl-Critic-Pulp-96
+perl-Perl-Tidy-20190601
+perl-Pod-MinimumVersion-50
+perl-Pod-POM-2.01
+perl-Pod-Spell-1.20
+perl-PPI-1.236
+perl-PPIx-Regexp-0.058
+perl-PPIx-Utilities-1.001000
+perl-Readonly-2.05
+perl-Regexp-Common-2017060201
+perl-Role-Tiny-2.000006
+perl-Scope-Guard-0.21
+perl-Selenium-Remote-Driver-1.33
+perl-Socket6-0.28
+perl-Socket-MsgHdr-0.04
+perl-Sort-Versions-1.62
+perl-Specio-0.42
+perl-SQL-Abstract-1.86
+perl-SQL-Translator-0.11024
+perl-strictures-2.000005
+perl-String-Format-1.18
+perl-String-ShellQuote-1.04
+perl-Sub-Exporter-0.987
+perl-Sub-Exporter-Progressive-0.001013
+perl-Sub-Identify-0.14
+perl-Sub-Install-0.928
+perl-Sub-Name-0.21
+perl-Sub-Quote-2.004000
+perl-Sub-Retry-0.06
+perl-Sub-Uplevel-0.240.0
+perl-SUPER-1.20141117
+perl-Task-Weaken-1.06
+perl-Test-Compile-1.3.0
+perl-Test-Exception-0.430000
+perl-Test-Fatal-0.014
+perl-Test-LongString-0.17
+perl-Test-MockModule-0.13
+perl-Test-MockObject-1.20161202
+perl-Test-Mock-Time-0.1.7
+perl-Test-Output-1.031
+perl-Test-Strict-0.40
+perl-Test-Warnings-0.026
+perl-Text-Brew-0.02
+perl-Text-Diff-1.45
+perl-TimeDate-2.30
+perl-Time-modules-2011.0517
+perl-Try-Tiny-0.30
+perl-Types-Serialiser-1.0
+perl-UNIVERSAL-can-1.20140328
+perl-UNIVERSAL-isa-1.20171012
+perl-UNIVERSAL-require-0.18
+perl-URI-1.73
+perl-URI-db-0.18
+perl-URI-Nested-0.10
+perl-Variable-Magic-0.62
+perl-WWW-RobotRules-6.02
+perl-XML-LibXML-2.0132
+perl-XML-NamespaceSupport-1.12
+perl-XML-SAX-0.99
+perl-XML-SAX-Base-1.09
+perl-XML-SAX-Expat-0.51
+perl-XML-SemanticDiff-1.0006
+perl-XML-Simple-2.24
+perl-YAML-1.24
+perl-YAML-LibYAML-0.80
+python-base-2.7.14
+ShellCheck-0.6.0

```

Comment:
now you managed to again duplicate all dependencies in another file. I can't approve that.

Reply:
this file is generated and maintained by bot. Discussed and agreed to with the team - sorry you had vacations

Reply:
ok, sorry. Is the relevant part of the discussion written down somewhere? I am ok with this part then but I wonder if there is a way where we can avoid having generated files next to human-maintained files. Or at least have an explanatory comment on the top like "# This is an auto-generated file, use script <foo> to update it if necessary"

Reply:
This is not about managing dependencies of OpenQA project - it is about making sure that list of dependencies (with versions) is valid, approved and consistent across tests in CI and optionally locally. (instead of relying on one container for each possible scenario)

Reply:
A plus for the "# This is an auto-generated file, use script to update it if necessary" comment.

Reply:
> A plus for the "# This is an auto-generated file, use script to update it if necessary" comment.

Yeah that can help, but will add some little work for stripping out comments when passing this list to zypper

Reply:
https://progress.opensuse.org/issues/57047